internal/test/test.go

```diff
@@ -55,43 +62,80 @@ func run(
 	_ command.GlobalFlags,
 	services *services.Services,
 ) (command.Result, error) {
+	testFiles := make(map[string][]byte, 0)
+	for _, filename := range args {
+		code, err := readerWriter.ReadFile(filename)
 
-	filename := args[0]
+		if err != nil {
+			return nil, fmt.Errorf("error loading script file: %w", err)
+		}
 
-	code, err := readerWriter.ReadFile(filename)
-	if err != nil {
-		return nil, fmt.Errorf("error loading script file: %w", err)
+		testFiles[filename] = code
+	}
+
+	var coverageReport *runtime.CoverageReport
```

Comment:
maybe better if we just pass the coverage report as boolean to the service execute function and there internally handle creation of `runtime.NewCoverageReport` to hide the implementation details from API layer

Reply:
Valid point :100:
Fixed in https://github.com/onflow/flow-cli/pull/827/commits/be87f5b054d99116ac836796def7c271c82174a9